### PR TITLE
Config parser

### PR DESCRIPTION
**Pull request by Pommfritz0308**

### CODE REVIEW

**Review by StrongCoding — DISMISSED** [review]
thanks for the fixes

File: src/Client.cpp [hunk 1]
```diff
@@ -153,7 +152,8 @@ void Client::readFromClient() {
 		size_t parsedSize = _header.parseBuffer(buffer);
 		if (parsedSize < static_cast<size_t>(bytes_received)) {
 			// save rest to bodybuffer
-			_bodyBuffer = std::string(&buffer[parsedSize]);
+			for (; parsedSize < static_cast<size_t>(bytes_received); parsedSize++)
+				_bodyBuffer.push_back(buffer[parsedSize]);
```
Comment (wrottger) [inline src/Client.cpp:156]: `_bodyBuffer = std::string(&buffer[parsedSize], parsed_size - bytes_received);`
this safes new allocations

File: src/HttpHeader.cpp [hunk 2]
```diff
@@ -94,7 +94,7 @@ bool HttpHeader::isInHeader(const std::string &name) const {
 // Returns true if the request is a Transfer-Encoding: chunked request
 bool HttpHeader::isTransferEncodingChunked() const {
 	std::map<std::string, std::string>::const_iterator it = message.headers.find("transfer-encoding");
-	return it != message.headers.end() && it->second.find("chunked") != std::string::npos;
+	return it != message.headers.end() && it->second.find("Chunked") != std::string::npos;
 }
```
Comment (wrottger) [inline src/HttpHeader.cpp]: it should be lowercase?

**Review by wrottger — CHANGES_REQUESTED** [review]

File: src/HttpHeader.cpp (hunk 2 above)
Comment (StrongCoding) [reply src/HttpHeader.cpp]: yes

**Review by wrottger — APPROVED** [review]
Looks good

**Pull request merged**